cumulus_etl/loaders/fhir/export_log.py

```diff
@@ -321,15 +321,15 @@ def download_error(self, file_url: str | httpx.URL, exc: Exception):
 
     def export_complete(self):
         timestamp = common.datetime_now(local=True)
-        duration = (timestamp - self._start_time) if self._start_time else 0
+        duration = (timestamp - self._start_time) if self._start_time else None
         self._event(
             "export_complete",
             {
                 "files": self._num_files,
                 "resources": self._num_resources,
                 "bytes": self._num_bytes,
                 "attachments": None,
-                "duration": duration.microseconds // 1000,
+                "duration": duration // datetime.timedelta(milliseconds=1) if duration else 0,
```

Comment:
Boneheaded mistake :roll_eyes: 

`.microseconds` is just the microseconds field, not the total time of the duration in microseconds.